.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - '**.md' #Do not need to run CI for markdown changes.
+  pull_request:
+    branches: [main]
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build-test:
+    strategy:
+      matrix:
+        versions: 
+        - {os: 'ubuntu-20.04', otp: '21.x', rebar: '3.15.2'}
```

Comment:
This runs fewer combinations than we were before. Part of that is that our elixir tests were just running things and elixir happened to be installed, but it didn't really test it working with elixir. I think longer term we want a better test for that. Like an elixir project that includes the SDK and we build that project.